webapp/src/app/services/modloader.service.ts

```diff
@@ -0,0 +1,63 @@
+import { Injectable } from '@angular/core';
+import {Globals} from '../shared/globals';
+import {GenericModLoader, Mod} from '@ac2pic/modloader';
+import {Observable} from 'rxjs';
+import {HttpClient} from '@angular/common/http';
+
+@Injectable({
+    providedIn: 'root'
+})
+export class ModloaderService {
```

Comment:
Is this service really needed? It would make a lot more sense to hide this code in http and move all relevant logic to `common` while consolidating with the code from `backend`